string action
string robot_type
string robot_name
string pose_type
string pose_name
float32 speed_scaling
float32 acc_scaling
float32 goal_tolerance